doc/topics/packaging/testing.rst

```diff
@@ -0,0 +1,150 @@
+``tools`` command suite that is located under ``<repo-root>/tools/testsuite/``.
+This method requires the Github CLI tool ``gh`` to be properly configured for
+interaction with the salt repo.
+
+#. Install the dependencies using this command:
+
+    .. code-block:: bash
+
+       pip install -r requirements/static/ci/py{python_version}/tools.txt
+
+#. Download and extract the artifacts with this ``tools`` command:
+
+
+    .. code-block:: bash
+
+        tools ts setup --platform {linux|darwin|windows} --slug
+        <operating-system-slug> --pr <pr-number> --pkg
+
+    The most common use case is to test the packages built on a CI/CD run for a
+    given PR. To see the possible options for each argument, and other ways to
+    utilize this command, use the following.
+
+    .. code-block:: bash
+
+        tools ts setup -h
+
+
+Downloading individually
+------------------------
+If the ``tools ts setup`` command doesn't work, you can download, unzip, and
+place the artifacts in the correct locations manually. Typically, you want to
+test packages built on a CI/CD run for a given PR. This guide explains how to
+set up for running the package tests using those artifacts. An analogous process
+can be performed for artifacts from nightly builds.
+
+#. Find and download the artifacts:
+
+    Under the summary page for the most recent actions run for that PR, there is
+    a list of available artifacts from that run that can be downloaded. Download
+    the package artifacts by finding
+    ``salt-<major>.<minor>+<number>.<sha>-<arch>-<pkg-type>``.  For example, the
+    amd64 deb packages might look like:
+    ``salt-3006.2+123.01234567890-x86_64-deb``.
+
+    The onedir artifact will look like
+    ``salt-<major>.<minor>+<number>.<sha>-onedir-<platform>-<arch>.tar.xz``. For
+    instance, the macos x86_64 onedir may have the name
+    ``salt-3006.2+123.01234567890-onedir-darwin-x86_64.tar.xz``.
+
+    .. note::
+
+        Windows onedir artifacts have ``.zip`` extensions instead of ``tar.xz``
+
+    While it is optional, it is recommended to download the ``nox`` session
+    artifact as well.  This will have the form of
+    ``nox-<os-name>-test-pkgs-onedir-<arch>``. The amd64 Ubuntu 20.04 nox
+    artifact may look like ``nox-ubuntu-20.04-test-pkgs-onedir-x86_64``.
+
+#. Place the artifacts in the correct location:
+
+    Unzip the packages and place them in ``<repo-root>/pkg/artifacts/``.
+
+    You must unzip and untar the onedir packages and place them in
+    ``<repo-root>/artifacts/``. Windows onedir requires an additional unzip
+    action. If you set it up correctly, the ``<repo-root>/artifacts/salt``
+    directory then contains the uncompressed onedir files.
+
+    Additionally, decompress the ``nox`` artifact and place it under
+    ``<repo-root>/.nox/``.
+
+Running the tests
+=================
+You can run the test suite can now be run if all the artifacts are in the
```

Comment:
**Nitpick (copy-pasta error, blocking):** "You can run the test suite **can now be run** if all the artifacts are in..."